code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -690,10 +690,12 @@
 	proc/cooltime()
 		if (src.warm)
 			spawn(4200)
+				if(src)
```

Comment:
Either this if doesn't do anything, or everything below it isn't tabbed correctly.

Reply:
Fixed.